### PR TITLE
Remove reference to SourceForge on footer #345

### PR DESCRIPTION
**Pull request by josecollado**
Changed the HTML a tag to open the github discussions link : https://github.com/jamulussoftware/jamulus/discussions
and also changed the logo img tag to a question mark font from https://icongr.am/fontawesome

### CODE REVIEW

**Comment by ann0see** [comment]
Great! Thank you.

**Review by ann0see — COMMENTED** [review]
Sorry, I forgot to tell you that this should go to the release branch directly ;-(. 

I‘ve just changed the branch this goes to release. I think this doesn’t break anything.

File: _includes/footer.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
     </a>
   </div>
   <div class="fx-col-100-xs fx-col-33-s fx-col-center-xs">
-    <a href="https://sourceforge.net/projects/llcon" target="_blank" rel="noreferrer">
-      <img src="https://sourceforge.net/sflogo.php?type=17&amp;group_id=158367" alt="{{ site.data.general.footer.alt.sourceforge }}" class="fx-center" id="sflogo" />
+    <a href="https://github.com/jamulussoftware/jamulus/discussions" target="_blank" rel="noreferrer">
+      <img src="https://icongr.am/fontawesome/question-circle-o.svg?size=35&color=currentColor" alt="{{ site.data.general.footer.alt.sourceforge }}" class="fx-center" id="sflogo" />
```
Comment (ann0see) [inline _includes/footer.html:16]: Note: I think we’ll need to edit the alt text too. (See the _data) folder. I can do this too, if you want.

File: wiki/en/en-Contribution.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ See the [Jamulus GitHub repo](https://github.com/jamulussoftware/jamulus) for fu
 
 * If you find a mistake, typo or something out of date (in any language) on the website, you can [open an issue on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues).
 
-* If you think some documentation or information is missing or can be improved, post about that on [the discussion forum](https://github.com/jamulussoftware/jamuluswebsite/discussions) so that it can be discussed first.
```
Comment (ann0see) [inline wiki/en/en-Contribution.md:20]: @gilgongo I think this also needs to be fixed in other languages?

Comment (gilgongo) [reply wiki/en/en-Contribution.md:20]: Unless we can temporarily do it with Google Translate?

ES: el foro de discusión
IT: il forum di discussione
FR: le forum de discussion
DE: das Diskussionsforum

I can put those in on the relevant languages .md on release?

Comment (ann0see) [reply wiki/en/en-Contribution.md:20]: German is wrong. Maybe translate the whole sentence?

Comment (hoffie) [reply wiki/en/en-Contribution.md:20]: Alternatively just replace the link in `release`, update English in `changes` and accept that the description is slightly incorrect until the next translation.

Comment (ann0see) [reply wiki/en/en-Contribution.md:20]: I think the link is still valid. I'll just merge this now.

**Review by ann0see — APPROVED** [review]

File: wiki/en/en-Contribution.md (hunk 2 above)
Comment (trebmuh) [reply wiki/en/en-Contribution.md:20]: _(the French translation is right)_

Comment (ignotus666) [reply wiki/en/en-Contribution.md:20]: Spanish is ok too

**Pull request merged**